### DISCUSSION TITLE
Build error

### DISCUSSION THREAD

**Discussion by jefbur** [ComplianceAsCode/content#9455] [Q&A] [answered]
When I try to build using a custom profile, I get the following error:

ValueError: Value 'var_accounts_minimum_age_login_defs' in profile 'test_rhel7' uses the selector '3'. This is not possible, as only selectors ['1', '2', '5', '7', 'default', '0'] are available. Either change the selector used in the profile, or add the selector-value pair to the variable definition.

If I really want to use 3 as the value, what do I need to modify to allow 3?

Thanks!

**Comment by yuumasato** [comment, marked as answer]
You need to add a new selector here:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-restrictions/password_expiration/var_accounts_minimum_age_login_defs.var

**Reply by jefbur** [reply]
Thanks!

**Comment by marcusburghardt** [comment]
I proposed new options for this variable: https://github.com/ComplianceAsCode/content/pull/11052

**Reply by marcusburghardt** [reply]
It is merged.
